CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## UNRELEASED
 
+- docs: add documentation for DpCheckbox
+- refactor: Move DpCheckbox to components root
```

Comment:
I think the changelog may not contain implementation changes. In this case, it does not matter if we move components around as long as they are exported from the top level `index.js`.

Reply:
removed in c5025cdc4b90074f4998b25225a33e8984550a3b